packages/console/src/assets/docs/guides/web-go/README.mdx

```diff
@@ -247,25 +235,22 @@ func main() {
 }`}
 </Code>
 
-Now, when your user visit `http://localhost:8080/sign-in`, the user will be redirected to the Logto sign-in page.
+Now, when your user visit `http://localhost:3000/sign-in`, the user will be redirected to the Logto sign-in page.
 
 </Step>
 
-<Step
-  title="Implement the callback route"
->
+<Step title="Implement the callback route">
 
 When the user signs in successfully on the Logto sign-in page, Logto will redirect the user to the Redirect URI.
 
-Assuming your Redirect URI is `http://localhost:8080/sign-in-callback`, then we will add the `/sign-in-callback` route to handle the callback after signing in.
+Assuming your Redirect URI is `http://localhost:3000/callback`, then we will add the `/callback` route to handle the callback after signing in.
```

Comment:
can you rebase master and use `defaultRedirectUri` exported from `_redirect-uris-web.mdx`?